spiceypy/tests/test_wrapper.py

```diff
@@ -6386,6 +6876,12 @@ def test_subpt():
     sep = spice.vsep(point1, point2) * spice.dpr()
     npt.assert_almost_equal(dist, 16.705476097706171)
     npt.assert_almost_equal(sep, 0.15016657506598063)
+    # Iterable ET argument to spice.subpt()
+    point1Alt1Arr = spice.subpt("near point", "earth", [et-20.,et,et+20.], "lt+s", "moon")
+    assert (3,2,) == point1Alt1Arr.shape
+    assert 0. == spice.vnorm(spice.vsub(point1Alt1Arr[1,0],point1))
```

Comment:
probably worth using pytest.approx(0.0) instead for all floating point comparisons like this.

Reply:
Suggest to decline:  if this is not exactly zero, then there is a problem, which is why I explicitly chose [0,0 == ...] (I almost used [... is 0.0;-]).

But still your call; let me know.